### PR TITLE
Allow qmake pattern to use `subdir`.

### PR DESCRIPTION
**Pull request by ahkok**

### CODE REVIEW

**Comment by ahkok** [comment]
@bryteise like this?

File: autospec/specfiles.py [hunk 1]
```diff
@@ -1259,17 +1259,26 @@ def write_qmake_pattern(self):
...
         self.write_variables()
 
+        if self.subdir:
+            self._write_strip("pushd " + self.subdir)
+
         self._write_strip("%qmake {}{}".format(extra_qmake_args, config.extra_configure))
         self._write_strip("test -r config.log && cat config.log")
         self.write_make_line()
         if config.config_opts['use_avx2']:
+            if self.subdir:
```
Comment (bryteise) [inline autospec/specfiles.py]: That'll work though I'd just do the
```
if self.subdir:
                self._write_strip("popd")
```
after the `self.write_make_line()` call directly and not inside the avx2 check. Saves from needing the else. Thoughts?

Comment (ahkok) [reply autospec/specfiles.py]: too many meetings today.

**Comment by ahkok** [comment]
""First Try!""

**Pull request merged**